server/data/seeds/users.js

```diff
@@ -5,6 +5,7 @@ const faker = require('faker');
 const createFakeUser = () => ({
   firstName: faker.name.firstName(),
   lastName: faker.name.lastName(),
+  userName: faker.internet.userName(),
```

Comment:
Same here we can use just username not userName

Reply:
dont change the faker function name that is part of faker. Bruce never used username in his code. I added it. I dont care which one we just need to agree. I will change it in my pr to username.